e2e/rbd_helper.go

```diff
@@ -801,8 +801,8 @@ func deletePool(name string, cephFS bool, f *framework.Framework) error {
 		// ceph osd pool delete myfs-metadata myfs-metadata
 		// --yes-i-really-mean-it
 		// ceph osd pool delete myfs-replicated myfs-replicated
-		// --yes-i-really-mean-it
-		cmds = append(cmds, fmt.Sprintf("ceph fs fail %s", name),
+		// --yes-i-really-mean-it674
```

Comment:
```suggestion
		// --yes-i-really-mean-it
```
This may have sneaked in by mistake ?